src/Symfony/Component/VarDumper/Resources/functions/dump.php

```diff
@@ -30,3 +30,16 @@ function dump($var, ...$moreVars)
         return $var;
     }
 }
+
+if (!function_exists('dd')) {
+    function dd($var, ...$moreVars)
```

Comment:
Is the name of the function something that could be reconsidered? I get that it's easy to type, but it is not descriptive at all. Something like `dumpDie()` would only be 5 more keystrokes (assuming no IDE with autocompletion is used), but would be a lot better in showing its intention.

Luckily, it looks like [Php Inspections (EA Extended)](https://github.com/kalessil/phpinspectionsea) will recognize this as a debug statement. Still, I think naming functions `dd` does not fit the Symfony philosophy.

Reply:
Only in this case it's imo DX microoptimalization, that is worth it as you may write it thousand times a day...

Reply:
I vote for `dumpAndExitPhpSession` personally. Much more descriptive. Seriously though, this is about developer convenience and happiness. It might not be descriptive at first, but once you know what is it, it makes a lot of sense and you'll save collectively many thousands of unnecessary keystrokes.

Reply:
1. This should **never** be committed => so other co-workers could not find it strange or not really descriptive 
2. In order to learn this new feature, you have to read the doc / blog post / this PR. 

So basically, the name is not so important. So, IMHO, we should keep it small and simple

Reply:
Looking at previous proposals, almost all of them proposed dd(). It feels like a strong indication this is the name we should settle for.